docs/observability/metrics.md

```diff
@@ -135,6 +138,9 @@ Used VM filesystem capacity in bytes. Type: Gauge.
 ### kubevirt_vmi_info
 Information about VirtualMachineInstances. Type: Gauge.
 
+### kubevirt_vmi_io_pressure_seconds
+Time tasks were delayed due to lack of IO resources. Type can be some or full (some is the walltime for some (one or more) tasks while full is the walltime for all tasks). Type: Counter.
```

Comment:
How is this different from the CPU IO wait metric?

Reply:
we already have a metric for the IO wait?